ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -123,8 +119,8 @@ export const FormContextProvider = ({
     setFormState(newFormState);
 
     if (currState.htmlType === 'file') {
-      const isFormValid = isFormStateValid(newFormState);
-      setHasValidationErrors(!isFormValid);
+      const validationErrorsList = getValidationErrors(newFormState);
```

Comment:
I was trying to work out if we could make this logic any simpler. E.g. don't love the duplicate if statements here.